doc/source/reference/apis/external-prediction.md

```diff
@@ -19,7 +19,57 @@ The Seldon Core exposes a generic external API to connect your ML runtime predic
 ### Feedback
 
  - endpoint : POST /api/v1.0/feedback
- - payload : JSON representation of ```Feedback``` - see [proto definition](./prediction.md#proto-buffer-and-grpc-definition)
+ - payload : JSON representation of `Feedback` - see [proto definition](./prediction.md#proto-buffer-and-grpc-definition)
+
+### Metadata - Graph Level (incubating)
```

Comment:
Great docs! I think we're at a point where we should add a top level metadata section, where all things from graph level to model level metadata is covered, and links to all the different areas are provided. Mainly as it currently it's quite hard to actually find where the metadata can be found. It woudl also be good to add a diagram for ituition, you can use this one (or a variation): https://docs.google.com/presentation/d/1-h-jYRarbEwpCIMeBxnDOgtvbLxMH1q6zyI3g6Ns4l4/edit#slide=id.g85303d6e24_0_294

Reply:
added top-level metadata section in latest commit